src/StdUtils.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
+
+import "./console2.sol";
+
+abstract contract StdUtils {
+    uint256 internal constant UINT256_MAX =
```

Comment:
I think this should be available from `CommonBase` instead from a specific Component.